.github/workflows/weekly_vuln_scan.yml

```diff
@@ -1,41 +1,35 @@
-# SPDX-FileCopyrightText: 2024 PNED G.I.E.
-# SPDX-License-Identifier: Apache-2.0
-
-name: Weekly Security Check
+name: Weekly Vulnerability Scan for Recent Images
 
 on:
   schedule:
-    - cron: "0 14 * * 1" # It will work at 14:00 on every Monday
-  workflow_dispatch:
+    - cron: "0 14 * * 1" # Her Pazartesi 14:00 UTC'de çalıştır
 
 jobs:
-  #ort:
-  # runs-on: ubuntu-latest
-  # steps:
-  #    - uses: actions/checkout@v4
-  #    - uses: oss-review-toolkit/ort-ci-github-action@7f23c1f8d169dad430e41df223d3b8409c7a156e
-  #      with:
-  #        allow-dynamic-versions: "true"
-  #        fail-on: "issues"
-  #        run: "cache-dependencies,cache-scan-results,labels,analyzer,evaluator,advisor,reporter,upload-results"
-
-  trivy:
+  vulnerability-scan:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      # Uncomment below to enable ORT job when needed
+      # - uses: oss-review-toolkit/ort-ci-github-action@7f23c1f8d169dad430e41df223d3b8409c7a156e
+      #   with:
+      #     allow-dynamic-versions: "true"
+      #     fail-on: "issues"
+      #     run: "cache-dependencies,cache-scan-results,labels,analyzer,evaluator,advisor,reporter,upload-results"
 
-      - name: Build Docker images
+      - name: Find Recent Docker Images
+        id: recent-images
         run: |
-          for Dockerfile in $(find . -name Dockerfile -mtime -7); do
-            image_name=$(dirname $Dockerfile | tr -d './')
-            docker build -t my-image-$image_name -f $Dockerfile .
-          done
+          FROM_DATE=$(date -d '7 days ago' '+%Y-%m-%d')
```

Comment:
The reference should not be time, but the last patch of the latest release or latest two, perhaps.

For example: if we have these branches `releases/v1.0`, `releases/v1.1`, and `releases/v2.0`, and theses tags `v1.0.0`, `v1.0.1`, `v1.1.0`, `v1.1.1`, `v1.1.2`, and `v2.0.0`, we should check only `v2.0.0` and `v1.1.2`.

Reply:
I am updating it to detect latest two ones, thanks for the example.